src/js/eventHandlers.js

```diff
@@ -13,8 +14,13 @@ function onPaginationClick(e) {
 
   const page = Number(paginationItem.dataset.id);
   queryParamsFilters.set('page', page);
+  queryParamsExercises.set('page', page);
 
-  rerender();
+  if (refs.exerciseHeader.dataset.exercise) {
+    rerenderExercises(queryParamsExercises);
```

Comment:
There is no longer any need to pass arguments, the service serviceGetExercises no longer accepts parameters.

Reply:
removed